src/components/10-atoms/toggle-switch/pw.ui.test.js

```diff
@@ -1,23 +1,44 @@
 const host = process.env.TEST_HOST_STORYBOOK_URL;
+const tag = 'axa-toggle-switch';
 
-it('should fire correct change event in controlled mode', async () => {
+describe('toggle-switch', () => {
+  it('should fire correct change event in controlled mode', async () => {
+    await openToggleSwitch();
+    await page.waitForSelector(tag);
+
+    const eventInfo = await page.textContent(
+      '.axa-toggle-switch-demo__event-info'
+    );
+    expect(eventInfo).toBe('-');
+    await page.click(tag);
+    const eventInfoAfterFirstClick = await page.textContent(
+      '.axa-toggle-switch-demo__event-info'
+    );
+    expect(eventInfoAfterFirstClick).toBe('true');
+    await page.click(tag);
+    const eventInfoAfterSecondClick = await page.textContent(
+      '.axa-toggle-switch-demo__event-info'
+    );
+    expect(eventInfoAfterSecondClick).toBe('true');
+  });
+
+  it('should write error message', async () => {
+    await openToggleSwitch();
+    await page.waitForSelector(tag);
+
+    await page.evaluate(evaluateTag => {
+      document
+        .querySelector(evaluateTag)
+        .setAttribute('error', 'Error Message');
+    }, tag);
+
+    const axaToggleSwitch = await page.$(tag);
+    expect(await axaToggleSwitch.getAttribute('error')).toBe('Error Message');
```

Comment:
This test is not very effective. Because the string is always at the component as attribute. You have to check if the dom element which shows the text (the span or the axa-text) is visible.

Reply:
Glad you have seen this, this test is actually pretty useless... 😄